### PR TITLE
Replace //external:protocol_compiler with //third_party:protocol_compiler

### PR DESCRIPTION
**Pull request by mering**
As `bind()` is not compatible with bzlmod, use the `alias()` targets instead.

### CODE REVIEW

**Comment by Wyverald** [comment]
Let's use the version `1.48.1.bcr.1`. Appending `.bcr.N` is the conventional way to submit a BCR-only patch.

**Comment by mering** [comment]
@Wyverald using `1.48.1.bcr.2` as `.1` already exists.

File: modules/grpc/metadata.json [hunk 1]
```diff
@@ -8,7 +8,8 @@
         "1.41.0",
         "1.47.0",
         "1.48.1",
-        "1.48.1.bcr.1"
+        "1.48.1.bcr.1",
+	"1.48.1.bcr.2"
```
Comment (Wyverald) [inline modules/grpc/metadata.json:12]: nit - could you adjust this to use spaces?

Comment (mering) [reply modules/grpc/metadata.json]: Done.

**Review by Wyverald — APPROVED** [review]

**Pull request merged**